pygeoapi/templates/base.html

```diff
@@ -40,6 +40,7 @@
         <a href="{{ config['server']['url'] }}">Home</a>
         {% endblock %}
         <span style="float:right">
+          <a href="{{ data['links'] | map(rel='self') | attr('href') }}?f=geopackage">GEOPACKAGE</a>
```

Comment:
I think this should be in the list of links in the main part of the page, rather than this "navbar" part. The JSON and JSON-LD representations can provide metadata about a collection, without any feature-level data. But the GeoPackage is intended for features, not metadata.

Reply:
Yes, agree. I can fix this in the PR. Thanks @alpha-beta-soup . I may throw in the ability to download the complete dataset as GeoPackage as well.